src/run_quac.py

```diff
@@ -231,20 +243,21 @@ def main(args):
     )
 
     # submit snakemake command as a slurm job
-    _, slurm_partition_times = read_workflow_config(args.workflow_config)
-
-    slurm_resources = {
-        "partition": args.slurm_partition,
-        "ntasks": "1",
-        "time": slurm_partition_times[args.slurm_partition],
-        "cpus-per-task": "1" if args.subtasks_slurm else "4",
-        "mem-per-cpu": "8G",
-    }
+    slurm_resources = {}
+    if args.cli_cluster_config:
+        if get_tool_path("sbatch") is None:
+            print(
+                f"ERROR: '--cli_cluster_config' was supplied to utilize SLURM, but SLURM's 'sbatch' tool was not found in your environment."
+            )
+            raise SystemExit(1)
+
```

Comment:
```suggestion

```
remove this in favor of adding dependency checks via above recommendation

Reply:
Done :)